Loop/Utilities/TranslationLayer.swift

```diff
@@ -0,0 +1,93 @@
+//
```

Comment:
Would recommend naming this `RectangleTranslationLayer` just in case we decide to implement more importers